pages/dkp/kommander/2.2/dkp-upgrade/upgrade-konvoy/essential/index.md

```diff
@@ -0,0 +1,166 @@
+---
+layout: layout.pug
+navigationTitle: DKP Essential
+title: Upgrade Konvoy for DKP Essential
+excerpt: Upgrade your Konvoy environment within the DKP Essential License.
+beta: false
+enterprise: false
+menuWeight: 30
+---
+## Prerequisites
+
+* Create an [on-demand backup][backup] of your current configuration with Velero.
+
+* Follow the steps listed in the [DKP upgrade overview][dkpup].
+
+* Ensure that all platform applications in the management cluster have been upgraded to avoid compatibility issues with the [Kubernetes version][releasenotes] included in this release. This is done automatically when [upgrading Kommander][upgradekomm], so ensure that you upgrade Kommander prior to upgrading Konvoy.
+
+* For air-gapped: Download the required bundles either at our [support site][supportsite] or by using the CLI.
+
+* For Azure, set the required [environment variables][envariables].
+
+* For AWS, set the required [environment variables][envariables2].
+
+* When upgrading, ensure that you set the flag `--etcd-version=3.4.13-0.`
+
+The following infrastructure environments are supported:
+
+* Amazon Web Services (AWS)
+
+* Microsoft Azure
+
+* Pre-provisioned environments
+
+## Overview
+
+To upgrade Konvoy for DKP Essential:
+
+1. Upgrade the Cluster API (CAPI) components
+1. Upgrade the core addons
+1. Upgrade the Kubernetes version
+
+If you have more than one Essential license, repeat all of these steps for each Essential cluster (management cluster). If an AMI was specified when initially creating a cluster, you must build a new one with [Konvoy Image Builder][KIB] and pass it with `--ami.`
```

Comment:
This is specific to AWS. Unfortunately, we don't have support for custom images in Azure. Let's move this to an AWS-specific note just next to the preprovisioned-specific note we have, just below.